src/main/java/org/tl/Hobbies.java

```diff
@@ -1,68 +1,174 @@
 package org.tl;
 
-import java.util.List;
-import java.util.Map;
+import java.io.*;
+import java.util.*;
 
 public class Hobbies {
 
+
     /**
      * Collect lines from the given file into a list.
      *
      * @param filePath The path to the file to read.
      * @return A list of lines from the file.
      */
     public List<String> createListFromFile(String filePath) {
-        return null;
+        List<String> lines = new ArrayList<>();
+        try {
+            File file = new File(filePath);
+            Scanner scanner = new Scanner(file);
+            while (scanner.hasNextLine()) {
+                String line = scanner.nextLine();
+                lines.add(line);
+            }
+
+            scanner.close();
+        } catch (FileNotFoundException e) {
+            System.out.printf("Error occurred %s%n", e.getMessage());
+        }
+        return lines;
     }
 
+
     /**
      * Create a dictionary of people's hobbies from the given file.
      *
      * @param filePath The path to the file to read.
      * @return A map where each person's name is associated with a list of their hobbies.
      */
     public Map<String, List<String>> createDictionary(String filePath) {
-        return null;
+        Map<String, List<String>> result = new HashMap<>();
+        List<String> lines = createListFromFile(filePath);
+        for (String line : lines) {
+            String[] parts = line.split(":");
+            String name = parts[0].trim();
+            String[] hobbies = parts[1].split(",");
+            List<String> hobbyList = new ArrayList<>(Arrays.asList(hobbies));
+            result.put(name, hobbyList);
+        }
```

Comment:
Not exactly correct, each person is going to have only one hobby because you are creating new hobby list each time and replacing the old value. In the tests it's useful to assert multiple things like making sure that a person with more than one hobby actually has more than one hobby. As a tip, you can leave 15-20 lines in the `hobbies.txt` file that is used in tests. This way it's more managable to make good asserts